README.md

```diff
@@ -13,45 +13,67 @@ Another application, [contacts-frontend](https://github.com/alphagov/contacts-fr
 
-## Dependencies
+## Technical documentation
 
-* Ruby 2.1.4
-* MySQL
-* Rummager (for GOV.UK site search)
+This is a Ruby on Rails application that serves 2 purposes:
 
-## Database setup
+1. It provides the public index of an organisation's contacts to users.
+2. It provides a private admin UI for signon users with permission to allow
+   them to manage the contacts for their organisation.
 
-The best way to get a database with good seed data is to use a dump from preview,
-alternatively you can load the database schema and use the old initial seed data:
+The app has its own database (MySQL).  The frontend part of this app reads
+contacts directly from this database.  This is in contrast to the separate [contacts-frontend](https://github.com/alphagov/contacts-frontend) app which reads contacts from the [content-store](https://github.com/alphagov/content-store).  The admin UI part of this app handles publishing the contacts to the [publishing-api](https://github.com/alphagov/publishing-api) and [rummager](https://github.com/alphagov/rummager) so that they are present for [contacts-frontend](https://github.com/alphagov/contacts-frontend) to read.
 
-    ```
-    bundle exec rake db:schema:load
-    bundle exec rake db:seed
-    bundle exec rake contacts:import_hmrc DATA_FILE=db/contact-records.csv
-    ```
-
-## Running
+### Dependencies
 
-Start the application with `./startup.sh` or use bowler.
-
-open http://contacts.dev.gov.uk/contact/hm-revenue-customs/
-open http://contacts.dev.gov.uk/admin
+* Ruby 2.1.4
+* MySQL
+* [Rummager](https://github.com/alphagov/rummager) - to publish contacts so
+  they can be found by GOV.UK site search
+* [Whitehall](https://github.com/alphagov/whitehall)
+  1. to access the organisations API and maintain parity with the organisations
+     managed in whitehall.
+  2. to access the world locations API to provide the correct list of countries
+     for the country dropdown in the address form
+* [signon](https://github.com/alphagov/whitehall) - for managing user auth in
+  the admin part
+* [static](https://github.com/alphagov/static) - for getting global GOV.UK
+  templates for the public part
+* [content-store](https://github.com/alphagov/content-store) - for doing some
+  command-line based tasks to withdraw and redirect a contact
+
+### Running the application
+
+`./startup.sh`
+
+This runs `bundle install` to install dependencies and runs the app on port `3051`.  When using the GOV.UK development VM it will be available at http://contacts-admin.dev.gov.uk/.  The public frontend lives at `/contact/hm-revenue-customs/` and the admin UI at `/admin`
 
 By default, this application uses the GOV.UK preview environment for assets. To
 run against a local version of static you need to set `STATIC_DEV` to
 "http://static.dev.gov.uk".
 
 
-## Development notes
+### Running the test suite
 
-* ```app/tasks``` - contains one-off tasks that can be run via console or a rake task.
-
-## Tests
+`bundle exec rake`
```

Comment:
I don't think so, no.  Rails should run that for you the first time; if you run `rake`, it won't do it if you run `rspec` on it's own though which is what the old readme suggested.  Made slightly confusing because `rake` will ultimately end up running `rspec` so the effect of running the commands should be the same, but `rake` will be marginally slower because it does a wee-bit more up-front, `rspec` just barrells straight on into running the test suite.

That said!  Might be worth trying to see if I'm correct or not.